src/handlers/dashboard/get_chart_data.py

```diff
@@ -112,17 +124,19 @@ def chart_data_handler(event, context):
     del context
     query_params = event["queryStringParameters"]
     filters = event["multiValueQueryStringParameters"].get("filter", [])
+    if "filter" in query_params and filters == []:
+        filters = [query_params["filter"]]
```

Comment:
Why the `filters == []` check (which could be `not filters` I think)? Do we want to ignore a `filter` query entirely if `multiValueQueryStringParameters` is set?

Reply:
in experimenting with this - I can't explictly say in cloudformation 'this parameter can be an array', so i'm not sure if it's going to be in `queryStringParameters` or `multiValueQueryStringParameters` in the AWS inbound event to the lambda. so I need to check both.

Reply:
if we got both (which I don't think should happen), `queryStringParameters` would have to be a subset of `multiValueQueryStringParameters`